App.js

```diff
@@ -31,6 +31,9 @@ import SupportScreen from './src/screens/SupportScreen';
 import TabBarIcon from './src/components/TabBarIcon';
 import Colors from './src/constants/Colors';
 
+import { useScreens } from 'react-native-screens';
+useScreens();
```

Comment:
What native navigation components did we need to utilize with this library?

Reply:
just these two lines